src/Scripting/ScriptingSystem.cpp

```diff
@@ -0,0 +1,62 @@
+#include "ScriptingSystem.h"
+#include "IBindings.h"
+
+#include <Utility/DataPath.h>
+#include <Library/Logger/Logger.h>
+
+#include <string>
+#include <vector>
+#include <memory>
+#include <utility>
+
+ScriptingSystem::ScriptingSystem(
+    std::string_view scriptFolder,
+    const std::vector<std::string> &entryPointFiles
+) : _entryPointFiles(entryPointFiles)
+  , _scriptFolder(scriptFolder) {
+    _initBaseLibraries();
+    _initPackageTable(scriptFolder);
+}
+
+std::unique_ptr<ScriptingSystem> ScriptingSystem::create(
+    std::string_view scriptFolder,
+    const std::vector<std::string> &entryPointFiles) {
```

Comment:
Style: just write it in a single line? We're in general comfy with lines up to 160 chars